tests/EventTest.php

```diff
@@ -165,6 +160,27 @@ public function testToArrayWithBreadcrumbs(): void
         $this->assertSame($breadcrumbs, $data['breadcrumbs']['values']);
     }
 
+    /**
+     * @dataProvider getMessageDataProvider
+     */
+    public function testGetMessage(array $message, array $expectedValue): void
```

Comment:
Please rename this variable to something that better reflect what it stores (e.g. `$setMessageMethodArguments`)